### PR TITLE
Update README.md

### LINKED ISSUE #731: Update README.md (scripts vs make)
The README.md is outdate. We no longer use make. It should be updated to explain how to use the scripts.

### PR DESCRIPTION
**Pull request by SleeplessByte**
Fixes #731

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,33 +3,130 @@
...
+```js
+/**
+ * Run this script (from root directory): npx @babel/node scripts/ci
+ *
+ * This will run following checks:
+ * 1. Check config in all exercises matches
+ * 2. Run eslint to check code-style
+ * 3. Run tests against sample solutions
+ */
+```
```
Comment (tejasbubane) [inline README.md]: These scripts are described in `CONTRIBUTING.md` (which is linked below) - I feel we don't need this elaborate description here, maybe have a shorter version to run `ci` command here and more details in contributing doc?

Comment (SleeplessByte) [reply README.md]: Yeah I like.

Comment (SleeplessByte) [reply README.md]: See push!

File: README.md [hunk 2]
```diff
@@ -3,33 +3,41 @@
...
-#### Test Specific Assignment
-Pass the exercise name to make script to run the tests for a specific exercise:
+## Contributing
 
-    make test-assignment ASSIGNMENT=hello-world
-    
-## Contributing Guide
-For an in-depth discussion of how exercism language tracks and exercises work, please see [CONTRIBUTING.md](https://github.com/exercism/javascript/blob/master/CONTRIBUTING.md)
+For an in-depth discussion of how exercism language tracks and exercises work, 
+please see [CONTRIBUTING.md][file-contributing].
```
Comment (tejasbubane) [inline README.md]: There are two mentions of `CONTRIBUTING.md`. Can you remove one of them?

Comment (SleeplessByte) [reply README.md]: I have it because people are _super_ lazy. But I guess we can remove this.

**Pull request merged**